frontend/src/features/settings/preferences/VideosPollUserSettingsForm.tsx

```diff
@@ -72,12 +104,71 @@ const VideosPollUserSettingsForm = () => {
   return (
     <form onSubmit={handleSubmit}>
       <Grid container spacing={4} direction="column" alignItems="stretch">
-        {/* Generic settings common to all polls */}
+        <Grid item>
+          <Typography variant="h6">
+            {t('pollUserSettingsForm.comparisonPage')}
+          </Typography>
+        </Grid>
+        {/*
+            Comparison UI: weekly collective goal display
+        */}
```

Comment:
Just a minor note. 

In the book Clean Code, Robert C. Martin describes comments as "Acknowledgements of failure to make the code readable" and he recommends to not have comments and if really comments are needed to be fully aware that they are failure to make the code readable.

I quite agree with this, even though it is quite extreme and it is very fine to not adhere to it 100%.

In this particular case, would the code be more readable with two separate components `CollectiveGoalDisplayFormSection` and `RateLaterAutoremovalFormSection`, then it would become obvious that the whole form is 

```
    <form onSubmit={handleSubmit}>
      <Grid container spacing={4} direction="column" alignItems="stretch">
        <Grid item>
          <Typography variant="h6">
            {t('pollUserSettingsForm.comparisonPage')}
          </Typography>
        </Grid>
        <CollectiveGoalDisplayFormSection {...someProps} />
        <RateLaterAutoremovalFormSection {...someProps} />
        <Grid item>
          <Button
            fullWidth
            type="submit"
            color="secondary"
            variant="contained"
            disabled={disabled}
          >
            {t('pollUserSettingsForm.updatePreferences')}
          </Button>
        </Grid>
      </Grid>
    </form>